### ISSUE TITLE
Building project

### ISSUE THREAD

**Issue by rafal-grycuk** [kriasoft/aspnet-starter-kit#32]
Hello, i have a problem with building application into the production. As far as i know, the output build should contain bundled css, js and html, which should be placed in the server. Unfortunately i cannot run build. Only the js files are created.
When i run ```npm run build``` i obtain error. I restored the packages but not it take no effect. The same problem is in "clean" project.
What seems to be the problem ? 
![2017_04_12_08_40_33_assets json_trading reactgrid_visual_studio_code](https://cloud.githubusercontent.com/assets/24410470/24944477/bcb05f2a-1f5b-11e7-8f43-9f699a642e00.png)

**Comment by ashutosh1211** [comment]
Getting same error while building for production :( using  `node run build`

**Comment by FluffyBucket** [comment]
Try replacing
`const args = ['publish', 'server', '-o', 'build', '-c', config, '-r', 'coreclr'];`
With
`const args = ['publish', 'server', '-o', '../build', '-c', config;`
In build task in run.js.
I got it working when replacing that, I am unsure however if you will still be using .net Core though.

**Comment by ashutosh1211** [comment]
or you can change the .cproj  and add RuntimeIdentifier manually

![image](https://cloud.githubusercontent.com/assets/4639147/26249602/e395f0b0-3cc4-11e7-95e7-fc1a90a0e7ed.png)

**Comment by FluffyBucket** [comment]
Yes, but when I did that I didn't get a server.dll file that can run.
And you will still need to change the `'build'` argument to `'../build'` since it will otherwise create a tree of build folders.

**Comment by ashutosh1211** [comment]
yes @FluffyBucket  you are right. `dotnet server.dll` is not running.
